app/views/spree/admin/payments/_form.html.erb

```diff
@@ -2,7 +2,7 @@
   <div class="alpha three columns">
     <div class="field">
       <%= f.label :amount, t(:amount) %>
-      <%= f.text_field :amount, :value => @order.outstanding_balance, :class => 'fullwidth' %>
+      <%= f.text_field :amount, :value => @order.outstanding_balance, :class => 'fullwidth', "watch-value-as" => 'form_data.amount', "ng-init" => "form_data.amount=#{@order.outstanding_balance}" %>
```

Comment:
Why `watch-value-as` instead of `ng-model`?

Reply:
This was rob's suggestion from the previous PR, as the data flow is only one way here.

Reply:
Do you need the `ng-init`? I was hoping the `watch-value-as` directive would take care of the initial assignment...